tests/tensor/test_extra_ops.py

```diff
@@ -482,6 +479,14 @@ def test_scalar_input(self):
         ):
             squeeze(x, axis=1)
 
+    def test_invalid_input(self):
```

Comment:
This can be merged with the new test above, it's the same graph and logic. You can compile the function once and test with the two inputs, valid and invalid